frontend/src/scenes/session-recordings/player/inspector/playerInspectorLogic.ts

```diff
@@ -721,4 +740,9 @@ export const playerInspectorLogic = kea<playerInspectorLogicType>([
             }
         },
     })),
+    events(({ actions }) => ({
+        afterMount: () => {
+            actions.loadMatchingEvents()
```

Comment:
This works but means if I change the filters, it doesn't react to the changes. I think we need to have some property listener for when the `matchingEventsMatchType` changes to trigger the re-calculation